From 903d2bee8c6f156790758f7e9131490b70475ecd Mon Sep 17 00:00:00 2001
From: Jiri Pirko <jiri@mellanox.com>
Date: Sat, 25 Nov 2017 11:07:57 +0100
Subject: [PATCH] tc: remove action cookie len from printout

Make the output same as input and avoid printout of unnecessary len.

Suggested-by: Stephen Hemminger <stephen@networkplumber.org>
Fixes: fd8b3d2c1b9b ("actions: Add support for user cookies")
Signed-off-by: Jiri Pirko <jiri@mellanox.com>
(cherry picked from commit 81051c60c24ad083cfcb46271e6450675763c722)
---
 tc/m_action.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tc/m_action.c b/tc/m_action.c
index 704708f70aaac..951ca973f4806 100644
--- a/tc/m_action.c
+++ b/tc/m_action.c
@@ -310,7 +310,7 @@ static int tc_print_one_action(FILE *f, struct rtattr *arg)
 		int strsz = RTA_PAYLOAD(tb[TCA_ACT_COOKIE]);
 		char b1[strsz * 2 + 1];
 
-		fprintf(f, "\tcookie len %d %s\n", strsz,
+		fprintf(f, "\tcookie %s\n",
 			hexstring_n2a(RTA_DATA(tb[TCA_ACT_COOKIE]),
 				      strsz, b1, sizeof(b1)));
 	}
-- 
2.13.1

